src/components/layout/header/index.tsx

```diff
@@ -1,5 +1,5 @@
 import { useState } from 'react';
```

Comment:
lets follow the convention set here and rename it to something else instead of `index.tsx`

https://deriv-group.slack.com/archives/C05Q4RDTR63/p1698809990190059

Reply:
this is @nada-deriv  component she already changed in her pr